lib/src/input_fields/primary_input_field.dart

```diff
@@ -88,14 +80,8 @@ class _MUIPrimaryInputField extends State<MUIPrimaryInputField> {
         style: widget.textStyle,
         obscureText: widget.isObscure,
         decoration: InputDecoration(
-          suffixIcon: IconButton(
-            onPressed: widget.suffixIconOnPressed,
-            icon: Icon(widget.suffixIcon),
-          ),
-          prefixIcon: IconButton(
-            onPressed: widget.prefixIconOnPressed,
-            icon: Icon(widget.prefixIcon),
-          ),
+          suffixIcon: widget.suffixIcon,
+          prefixIcon: widget.prefixIcon,
```

Comment:
Please don't let users to implement whole icon button instead what you can do...

You can stick with your previous implementation and only give IconButton if `widget.suffixIcon` is not null...

```dart
decoration: InputDecoration(
prefixIcon: widget.prefixIcon != null ? IconButton(...) : null,
suffixIcon: widget.suffixIcon != null ? IconButton(...) : null,
)
```